provisioning/roles/gulp/templates/gulp.config.js

```diff
@@ -1,32 +1,32 @@
-'use strict';
+const argv = require('yargs').argv;
 
 module.exports = {
+  optimize:         argv.production,
   src: {
     sass:           'static/sass/**/*.scss',
-{% if gulp_use_webpack %}
-    javascripts:    'static/javascripts/src/**/*.{js,jsx{% if gulp_use_purescript %},purs{% endif %}}',
-    webpack:        ['./static/javascripts/src/main.js'],
-{% endif %}
     images:         'static/images/**/*.{gif,jpg,jpeg,png,svg}',
     templates:      '**/*.html'
   },
   dest: {
     css:            'static/stylesheets',
-{% if gulp_use_webpack %}
-    webpack:        { path: './static/javascripts/', filename: 'main.js' },
-{% endif %}
     images:         'static/images'
   },
   browserSync: {
     proxy:          '{{ hostname }}',
+{% if ssl_key_file is defined %}
```

Comment:
That’s what I’ve done at first, but if you set `ssl: true` it doesn’t mean the `ssl` role will be installed (and certificates created) because it always depends on a webserver role. I’ve done my testing without any webserver, that’s why I run into this issue. Maybe its a bit too defensive since there’s not much reason somebody wouldn’t install any webserver?